src/cdk/a11y/aria-describer/aria-describer.ts

```diff
@@ -148,49 +168,54 @@ export class AriaDescriber implements OnDestroy {
     }
 
     this._createMessagesContainer();
-    messagesContainer!.appendChild(messageElement);
-    messageRegistry.set(getKey(message, role), {messageElement, referenceCount: 0});
+    this._messagesContainer!.appendChild(messageElement);
+    this._messageRegistry.set(getKey(message, role), {messageElement, referenceCount: 0});
   }
 
   /** Deletes the message element from the global messages container. */
   private _deleteMessageElement(key: string | Element) {
-    const registeredMessage = messageRegistry.get(key);
-    registeredMessage?.messageElement?.remove();
-    messageRegistry.delete(key);
+    this._messageRegistry.get(key)?.messageElement?.remove();
+    this._messageRegistry.delete(key);
   }
 
   /** Creates the global container for all aria-describedby messages. */
   private _createMessagesContainer() {
-    if (!messagesContainer) {
-      const preExistingContainer = this._document.getElementById(MESSAGES_CONTAINER_ID);
+    if (this._messagesContainer) {
```

Comment:
An internal test broke because the tooltip's message was changed and they wanted to confirm the aria-describeby was accurate. After the tooltip's message is changed, the container was removed but not set to null, so when it tried to set up the new message, this just returned.

Fix: When calling `this._messagesContainer?.remove()`, also set it to null.

Reply:
Done.